lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -116,7 +116,7 @@ class LoadFastEnough4Pwa extends Audit {
           return {
             rawValue: false,
             // eslint-disable-next-line max-len
-            debugString: `First Interactive was at ${timeToFirstInteractive.toLocaleString()}. More details in the "Performance" section.`,
+            debugString: `First Interactive was at ${timeToFirstInteractive.toLocaleString()}ms. More details in the "Performance" section.`,
```

Comment:
we separate numbers and unit with `\xa0` these days.

but you might as well just use `Util.formatMilliseconds` which will round it to the tens place, fixing your other concern with these numbers. :)